p2p/netutil/net.go

```diff
@@ -22,21 +22,19 @@ import (
 	"errors"
 	"fmt"
 	"net"
-	"sort"
+	"net/netip"
+	"slices"
 	"strings"
+
+	"golang.org/x/exp/maps"
```

Comment:
there is no this package in the go.mod

Reply:
No, I mean, it should be in the require block

Reply:
https://github.com/ethereum/go-ethereum/blob/4461c1fc17c489f1df86dea1ebde4247c6305ea4/go.mod#L142

Reply:
Run `go mod tidy` please

Reply:
#29941